Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,7 +539,7 @@ def audit_specs
       end
 
       if @new_formula && formula.head
-        problem "New formulae should not have a HEAD spec"
```

Comment:
@ilovezfs Is this what you're wanting? I wasn't sure.

Reply:
Yes. The original wording was spelling out exactly how to cheat the audit. Open a new formula, get it merged. Then immediately open a second PR to add head.